osrdyne/src/drivers/kubernetes.rs

```diff
@@ -298,9 +301,24 @@ impl KubernetesDriver {
                     type_: "rabbitmq".to_string(),
                     use_cached_metrics: Some(keda.use_cached_metrics),
                     metric_type: keda.metric_type.clone(),
+                    authentication_ref: {
+                        // If there is an authentication reference, we use it. Otherwise, we don't set this field.
+                        if let Some(auth_ref) = &keda.authentication_ref {
+                            Some(keda::AuthenticationRef {
+                                name: auth_ref.clone(),
+                            })
+                        } else {
+                            None
+                        }
```

Comment:
This could be 
```rust
keda.authentication_ref.map(|auth_ref| keda::AuthenticationRef {
                                name: auth_ref.clone(),
                            })
```

or something similar